src/core/create_compilers/RollupCompiler.ts

```diff
@@ -161,7 +161,10 @@ export default class RollupCompiler {
 		const that = this;
 
 		// TODO this is hacky. refactor out into an external rollup plugin
-		(mod.plugins || (mod.plugins = [])).push(css_chunks());
+		(mod.plugins || (mod.plugins = [])).push(css_chunks({
+			chunkFileNames: '[name]-[hash].css',
```

Comment:
this line shouldn't be necessary since it's the default

Reply:
I agree, deleted it.